google/cloud/storage/README.md

```diff
@@ -44,7 +44,38 @@ Apache 2.0; see [`LICENSE`](../../../LICENSE) for details.
 
 ## Release Notes
 
-### v1.4.x - TBD
+### v1.5.x - TBD
+
+### v1.4.x - 2019-09
+
+* feat: Random CRC and MD5 in storage throughput benchmark (#2943)
+* feat: Make GCS throughput benchmark record progress. (#2944)
+* feat: Increase download and upload buffers. (#2945)
+* feat: Increase the threshold fur using resumable uploads (#2946)
+* cleanup: Don't include nljson.h from public oauth2 headers. (#2954)
+* bug: Handle CURLE_{RECV,SEND}_ERROR as StatusCode::kUnavailable. (#2965)
+* cleanup: Use ObjectMetadata in ResumableUploadResponse. (#2969)
+* feat: Make integration tests client configurable. (#2970)
```

Comment:
Cloud remove this, but probably does not matter.

Reply:
Done.